vcd/resource_vcd_org_user.go

```diff
@@ -0,0 +1,352 @@
+package vcd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+)
+
+func resourceVcdOrgUser() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceVcdOrgUserCreate,
+		Read:   resourceVcdOrgUserRead,
+		Delete: resourceVcdOrgUserDelete,
+		Update: resourceVcdOrgUserUpdate,
+		Importer: &schema.ResourceImporter{
+			State: resourceVcdOrgUserImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validateCase("lower"),
+				Description:  "User's name. Only lowercase letters allowed. Cannot be changed after creation",
+			},
+			"org": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "Organization this user belongs to",
+			},
+			"role": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    false,
+				Description: "Role within the organization",
+			},
+			"password": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: false,
+
+				ConflictsWith: []string{"password_file"},
+				Description: "The user's password. This value is never returned on read. " +
+					`Either "password" or "password_file" must be included on creation.`,
+			},
+			"password_file": &schema.Schema{
+				Type:          schema.TypeString,
+				Optional:      true,
+				ForceNew:      false,
+				ConflictsWith: []string{"password"},
+				Description: "Name of a file containing the user's password. " +
+					`Either "password_file" or "password" must be included on creation.`,
+			},
+			"description": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    false,
+				Description: "The Org User's description",
```

Comment:
Capitalization: you're using "user" or "User" in different places. Please choose one and use everywhere.

Reply:
fixed